### DISCUSSION TITLE
EventSource - when is it coming?

### DISCUSSION THREAD

**Discussion by proddy** [alovajs/alova#318] [Q&A] [answered]
Hi, when is SSE/EventSource planning to be implemented? I could really use it and got excited when I saw this page: https://alova.js.org/zh-CN/tutorial/strategy/useSSE/

**Comment by JOU-amjs** [comment]
Sorry. Due to the limitation of time recently, I'm not sure when to implement it. maybe one month or two months later.
Once completed, I will notify you here.

**Comment by MeetinaXD** [comment, marked as answer]
👋 Hello, this feature has been implemented in `alova/scene@1.5.0`. You can update your scene package and try it .

BTW, the English version of the documentation has also been updated. 

Docs view here: [https://alova.js.org/tutorial/strategy/useSSE/](https://alova.js.org/tutorial/strategy/useSSE/)

**Reply by proddy** [reply]
that is awesome! can't wait to try. Thanks!!!

**Reply by proddy** [reply]
It works great. I replaced my React code
```js
  useEffect(() => {
    const es = new EventSource(LOG_EVENTSOURCE_URL);
    es.onmessage = (event: MessageEvent) => {..}
    es.onerror = () => {...} 
  }, []);
```
with useSSE, made some tweaks and it all works the same. Now I'm 100% Alova.

However it did add an additional 3kb to my bundle size which is not great since I'm running the code on a microcontroller with 4MB flash mem and space is precious

**Reply by MeetinaXD** [reply]
I'm glad this is helping you with your work.
I didn't expect alova to run on a microcontroller, though, and obviously some code optimization is needed in such an environment.
This will take some (maybe more) time, I will get back to you as soon as I have something.
